entrypoint_debian.sh

```diff
@@ -1,46 +1,27 @@
 #!/bin/bash -x
-
-# Karaf needs a pseudo-TTY so exit and instruct user to allocate one when necessary
-test -t 0
-if [ $? -eq 1 ]; then
-    echo "Please start the openHAB container with a pseudo-TTY using the -t option or 'tty: true' with docker compose"
-    exit 1
-fi
-
 set -euo pipefail
 IFS=$'\n\t'
 
-# Deleting instance.properties to avoid karaf PID conflict on restart
-# See: https://github.com/openhab/openhab-docker/issues/99
-rm -f /openhab/runtime/instances/instance.properties
-
-# The instance.properties file in OH2.x is installed in the tmp
-# directory
-rm -f /openhab/userdata/tmp/instances/instance.properties
-
 # Add openhab user & handle possible device groups for different host systems
 # Container base image puts dialout on group id 20, uucp on id 10
 # GPIO Group for RPI access
 NEW_USER_ID=${USER_ID:-9001}
-NEW_GROUP_ID=${GROUP_ID:-$NEW_USER_ID}
-echo "Starting with openhab user id: $NEW_USER_ID and group id: $NEW_GROUP_ID"
+echo "Starting with openhab user id: $NEW_USER_ID"
 if ! id -u openhab >/dev/null 2>&1; then
-  echo "Create group openhab with id ${NEW_GROUP_ID}"
-  groupadd -g $NEW_GROUP_ID openhab
   echo "Create user openhab with id ${NEW_USER_ID}"
-  adduser -u $NEW_USER_ID --disabled-password --gecos '' --home ${APPDIR} --gid $NEW_GROUP_ID openhab
-  groupadd -g 14 uucp2
-  groupadd -g 16 dialout2
-  groupadd -g 18 dialout3
-  groupadd -g 32 uucp3
-  groupadd -g 997 gpio
-  adduser openhab dialout
-  adduser openhab uucp
-  adduser openhab uucp2
-  adduser openhab dialout2
-  adduser openhab dialout3
-  adduser openhab uucp3
-  adduser openhab gpio
+  adduser -u $NEW_USER_ID --disabled-password --gecos '' --home ${APPDIR} openhab &&\
+    groupadd -g 14 uucp2 &&\
+    groupadd -g 16 dialout2 &&\
+    groupadd -g 18 dialout3 &&\
+    groupadd -g 32 uucp3 &&\
+    groupadd -g 997 gpio &&\
+    adduser openhab dialout &&\
+    adduser openhab uucp &&\
+    adduser openhab uucp2 &&\
+    adduser openhab dialout2 &&\
+    adduser openhab dialout3 &&\
+    adduser openhab uucp3 &&\
```

Comment:
These lines were unchained in https://github.com/openhab/openhab-docker/pull/142.